frontend/src/__tests__/api.test.js

```diff
@@ -54,6 +54,88 @@ expect.extend({
     }
 });
 
+/**
+ * Seeding the database with the minimal set of API calls to create an assignment.
+ */
+function dataBaseSeed(api = { apiGET, apiPOST }) {
+    const { apiGET, apiPOST } = api;
+    let session = null;
+
+    const minimalSessionData = {
+        start_date: new Date("2020-02-10").toISOString(),
+        end_date: new Date("2020-12-31").toISOString(),
+        name: "Initial Session",
+        rate1: 50
+    };
+
+    const minimalApplicantData = {
+        utorid: "cole",
+        student_number: "10000000",
+        first_name: "Cole",
+        last_name: "Zemel",
+        email: "cole.zemel@gmail.com",
```

Comment:
Is this a real email? It looks too real for comfort.

Reply:
No it's not but I'll change it